crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -470,6 +472,16 @@ define_tedge_config! {
         #[doku(as = "PathBuf")]
         root_cert_path: Utf8PathBuf,
 
+        /// The type of authentication method
+        #[tedge_config(example = "certificate", example = "basic", example = "auto", default(variable = RegistrationMode::Certificate))]
+        registration_mode: RegistrationMode,
```

Comment:
The term `registration_mode` is not crystal clear.

```suggestion
        /// The authentication method used to connect Cumulocity
        #[tedge_config(note = "In the auto mode, basic auth is used if c8y.credentials_path is set")]
        #[tedge_config(example = "certificate", example = "basic", example = "auto", default(variable = RegistrationMode::Certificate))]
        auth_method: RegistrationMode,
```

Reply:
Done in [6d8a324](https://github.com/thin-edge/thin-edge.io/pull/3196/commits/6d8a3241a80521d15e9215e1358b8930314215a0)